inc/Integrations/Airtable/AirtableDataSource.php

```diff
@@ -17,6 +17,7 @@ protected static function get_service_config_schema(): array {
 				'id' => Types::string(),
 				'name' => Types::nullable( Types::string() ),
 			] ),
+			'enable_blocks' => Types::nullable( Types::boolean(), true ),
```

Comment:
This second arg (`true`) doesn't do anything. Were you trying to set a default value? Also, this should trigger a lint / type error. 🤔

Reply:
> Were you trying to set a default value? 

Yes I was. I probably don't have a proper lint setup for PHP, but then I guess our tests didn't catch it either